### DISCUSSION TITLE
Zed Needs to Improve UI. Create a Design Pattern.

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#8763] [Show and tell]
The Zed editor should improve its UI. Perhaps it is simplified due to performance, however, it would be good if it combined the good of both worlds.
Performance + Design

## Font, File Tree and Menu

Currently, I don't like the font used in the editor menu and other parts of the editor - which should only be used in the code part (Zed Mono only for code, Calibri or Roboto for Menu, Windows, etc).
Regarding the file tree, I don't have many criticisms, but it could be improved by adding gray icons of the language logos. 
And menu is so simple. 

## Modern, Minimalist, Consistent

However, modern should not mean "more things", but rather something minimalist, like Flat Design (Apple specializes in creating designs of this type).
Zed must be the "opposite" of VSCode which added countless icons, a horrible sidebar, inconsistent design (colors, windows, fonts, layout) that visually polluted the editor.

See editors like Atom:

![images (4)](https://github.com/zed-industries/zed/assets/161986636/17c61b58-30ec-4c99-8421-77ef9bfe7928)

And Sublime Text:
![images (6)](https://github.com/zed-industries/zed/assets/161986636/038e76af-3007-404b-bfec-c5ec39271dba)

The two created a clean design pattern, with fluid animations, correct layout, easy to interact with, consistent and (It's clear) modern. However, one has been archived and the second isn't free...
I think that recycling some design features from other editors that are nice to look at (Sub. Text, Atom, Playcode.io for example) would be very efficient to make a design for Zed.

**Comment by Moshyfawn** [comment]
In #8499, I think this discussion is already happening.

**Reply by ghost** [reply]
I understand... but when I say improving the UI, it doesn't just mean modernizing it, but also creating a consistent design pattern - like Apple does in its software.
It is necessary for Zed to be pleasant to programmers, with an unpolluted look. Few code editors respect this side and focus more on the functionality part, which is not wrong, but they leave aside the OCD of code writers - Hello VSCode and Notepad++...

I think that recycling some design features from other editors that are nice to look at (Sub. Text, Atom, Playcode.io for example) would be very efficient.

**Reply by Moshyfawn** [reply]
Oh, I'm not trying to argue with you or shut down your idea of the design system, just point you to a conversation you might be interested in 😉

**Reply by ghost** [reply]
oh, sorry if I was a bit rude, I was trying to be direct 😅 Thanks

**Reply by emersonjds** [reply]
I agree with @Moshyfawn, there must be a design pattern so that new contributors can help improve the tool.

As mentioned in other features I opened, the developer experience must be fluid and efficient.

DX is extremely important in everyday life.

We use ZED for everything it can provide us in the future and not just now in the present, pay attention to this request from the community
